apps/synt-person-api/src/main/java/no/nav/registre/testnorge/synt/person/config/SwaggerConfig.java

```diff
@@ -0,0 +1,64 @@
+package no.nav.registre.testnorge.synt.person.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.MediaType;
+import org.springframework.web.servlet.config.annotation.ViewControllerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import springfox.documentation.annotations.ApiIgnore;
+import springfox.documentation.builders.ApiInfoBuilder;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.service.ApiInfo;
+import springfox.documentation.service.Contact;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Configure automated swagger API documentation
+ */
+
+@Configuration
+@EnableSwagger2
+public class SwaggerConfig implements WebMvcConfigurer {
+
+    @Value("${application.version}")
+    private String appVersion;
+
+    @Bean
+    public Docket api() {
+        Set<String> contentTypeJson = new HashSet<>(Collections.singletonList(MediaType.APPLICATION_JSON_VALUE));
+        return new Docket(DocumentationType.SWAGGER_2)
+                .ignoredParameterTypes(ApiIgnore.class)
+                .select()
+                .apis(RequestHandlerSelectors.any())
+                .paths(PathSelectors.ant("/api/**"))
+                .build()
+                .apiInfo(apiInfo())
+                .produces(contentTypeJson)
+                .consumes(contentTypeJson)
+                .useDefaultResponseMessages(false);
+    }
+
+    private ApiInfo apiInfo() {
+        return new ApiInfoBuilder()
+                .title("Testnorge Synt Arbeidsforhold API")
+                .description("API for syntetisering av arbeidsforhold")
```

Comment:
Oppdater swagger beskrivelse. Syntetisering av arbeidsforhold er vel ment for `synt-arbeidsforhold-api`?

Reply:
Copy paste... Takk